packages/devtools_app/lib/src/inspector/flutter/inspector_data_models.dart

```diff
@@ -205,3 +412,85 @@ class FlexLayoutProperties extends LayoutProperties {
   static final _textBaselineUtils =
       EnumUtils<TextBaseline>(TextBaseline.values);
 }
+
+class RenderInfo {
+  RenderInfo(this.axis, Size size, Offset offset, Size realSize)
```

Comment:
probably cleaner to just store
this.size, this.offset, and this.realSize directly instead of storing the components from them.

Reply:
nm. this is reasonable given how you are modifying it.